cli/services/config.js

```diff
@@ -72,10 +72,10 @@ function getAliases () {
       }
     })
     .sort(function (a, b) {
-      if (a.absPath === b.absPath) {
+      if (a.relPath === b.relPath) {
         return 0
       }
-      return a.absPath > b.absPath ? -1 : 1
+      return a.relPath > b.relPath ? -1 : 1
     })
```

Comment:
I changed this to use `relPath` instead of `absPath` so `../../` would be sorted after `../` regardless of parent directory names (as per `tests/specs/cli/get-aliases.spec.js > cli alias configuration > alias lookups should be in reverse path order`). This shouldn't be breaking, unless maybe folks are using symlinks or defining bloated aliases like `src/../src/...`?

Reply:
It's most likely to do with the CLI functionality, but it's a couple of years since I looked at that code so I'd have to dig in and look at what it does to the code again. 

From memory it's basically to ensure that the most specific alias is used when converting source code:

https://github.com/davestewart/alias-hq/blob/main/tests/specs/cli/update-source.spec.js#L82-L85

If that still ran, then we should be OK.

I'll take a look at #45 and consider it all together.

Thanks for doing the brainwork!

Reply:
> If that test still passed, then we should be OK.

It does :+1: